routes/movies.routes.js

```diff
@@ -0,0 +1,69 @@
+const Celebrity = require("../models/Celebrity.model");
+const { populate } = require("../models/Celebrity.model");
+const Movie = require("../models/Movie.model");
+
+const router = require("express").Router();
+
+router.get("/movies/create", (req, res) => {
+
+   Celebrity.find()
+    .populate("name")
+    .then( (celebrityArr) => {
+    
+    res.render("movies/new-movie", {celebrityArr});
+  })
+  })
+  
+  router.post("/movies/create", (req, res) => {
+    // console.log(req.body)
+    const movieDetails = {
+      title: req.body.title,
+      celebrity: req.body.celebrity,
+      plot: req.body.plot,
+      genre: req.body.genre,
+    };
```

Comment:
At the moment when you create your movie, you are not saving celebrities  in the database. 

Very important: Your model is representation of exactly how data is saved in the database.
Here is your movie model:

```
const movieSchema = new Schema(
 {
 title: {
        type: String,
        required: true,
    },
    genre: String,
    plot: {
        type: String,
        required: true,
  },
    cast: [{
        type: Schema.Types.ObjectId,
        ref: "Celebrity"
}]
});
```

You have property CAST - and that is what database is expecting. However when you are creating your movie via form -> you are using property name CELEBRITY